lib/puma/const.rb

```diff
@@ -124,7 +124,7 @@ module Const
     # we amortize the cost of going back to the reactor for a well behaved
     # but very "greedy" client across 10 requests. This prevents a not
     # well behaved client from monopolizing the thread forever.
-    MAX_FAST_INLINE = 10
+    MAX_FAST_INLINE = ENV.fetch('MAX_FAST_INLINE', 10).to_i
```

Comment:
Rather than have a constant set by an ENV variable, I think we should add this to the configuration object and then read it from there.